Geospatial/Utilities.cs

```diff
@@ -0,0 +1,23 @@
+﻿using OSGeo.OGR;
+using OSGeo.OSR;
+
+namespace Geospatial;
+public class Utilities
+{
+  public static void StructureFieldTypes(Feature feature, string fieldName, object value)
```

Comment:
also if for some reason nsi changes a field name - from fd_id to id or oid, it would break this code. i like the field type delegate and whatnot, but it is in a completely different namespace too - nothing about this is specifically geospatial, you could get a fieldname type key value pair collection dictionary or hash from the consequences core library and apply it to the feature - i just get nervous seeing string representations in more than one place in the code.

Reply:
I used reflection in the reader because I'm dealing with ConsequenceReceptors, here I am writing results so I was a bit more unsure. I understand the comment about having string representations in multiple places but not sure how to combat that here

Reply:
I considered moving the StructureFieldTypes elsewhere but it needs to operate on a layer so it had to stay in Geospatial

Reply:
I was suggesting maybe removing the field from the arguments of the method. and return a hash, list or dictionary of key value pairs of field name and type. that way you could define the info in core, and use it in geospatial - it isnt a hill worth dying on, just trying to think about how to best separate concerns.